projects/net-snmp/agentx_parse_fuzzer.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 Google LLC
+ * Copyright 2018 Google Inc.
```

Comment:
Please combine the previous commit and this commit into a single commit, e.g. as follows: `git reset HEAD^ && git commit -a --amend --no-edit`. Or alternatively, use `git rebase -i`.

Reply:
We will squash this commit when it gets merged, so it will all be a single commit in the oss-fuzz history.

Reply:
Who are "we"? Please squash the commits now to make it easier to review the proposed changes.

Reply:
Squashed it now! We just referred to whoever was going to merge the pull request in as I sometimes do that too.